### PR TITLE
updated installer with dhcp-range option

### PR DESCRIPTION
**Pull request by sghai**
`DHCP_RANGE` environment variable is defined with some specific range in jenkins.

### CODE REVIEW

**Comment by sghai** [comment]
Covers #270

**Comment by sghai** [comment]
@kbidarkar : Please take a look and suggest if any changes are required.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -250,6 +250,11 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         print('Was not possible to fetch domain information')
         sys.exit(1)
 
+    dhcp_range = os.environ.get('DHCP_RANGE')
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: Ok, so DHCP_RANGE is updated with the valid ranges.

**Comment by kbidarkar** [comment]
ACK

**Comment by sthirugn** [comment]
ACK

File: automation_tools/__init__.py (hunk 1 above)
Comment (omaciel) [inline automation_tools/__init__.py]: Wait. so we're not required to provide it? What if I don't want to do it? Couldn't we make it optional? << @kbidarkar @sghai

**Comment by sthirugn** [comment]
@omaciel I didnt notice your comment before merging, sorry about that.

**Comment by omaciel** [comment]
@sghai would you mind sending another PR which would make providing the DHCP_RANGE optional?

**Comment by elyezer** [comment]
@omaciel @sthirugn @sghai created #272 to make it optional.

**Comment by kbidarkar** [comment]
Note to self, while reviewing, I should also make a point to read/view the entire code. Looking at the changes it looks good but may be altering the code behavior.

**Comment by kbidarkar** [comment]
:+1:  @omaciel, @elyezer  for finding it and the fix. :)

**Pull request merged**